index.html

```diff
@@ -0,0 +1,333 @@
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <footer class="main">
+      <section id="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="https://formspree.io/f/xwkajypj" id="form" method="post">
+          <input
+            name="Name"
+            id="Name"
+            type="text"
+            placeholder="Full Name"
+            maxlength="30"
+          /><br />
+          <input
+            name="Email"
+            id="email"
+            type="text"
+            required
+            placeholder="Email address"
+          /><br />
+          <textarea
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            placeholder="Your Message"
+          ></textarea
+          ><br />
+          <button type="submit" class="submitButton">Get in touch</button>
```

Comment:
- [ ] Kindly modify the submit button to adhere to the design of the Figma template. The below screenshot is the correct design. 

![Correct submit button desing](https://user-images.githubusercontent.com/87413774/163391106-974951a6-0fb2-4208-869b-2dc628e7f343.png)

- [ ] The below screenshot is your implementation

![Wrong submit form design](https://user-images.githubusercontent.com/87413774/163391362-dd216358-73d4-4bc0-b359-c02aadd31318.png)

Reply:
Kindly make the submit button to be smaller and align to the left as it is in the Figma Design.